EasyPost/BetaFeatures/Parameters/Shipments/Create.cs

```diff
@@ -0,0 +1,48 @@
+using System.Collections.Generic;
+using EasyPost.Utilities.Internal.Annotations;
+
+namespace EasyPost.BetaFeatures.Parameters.Shipments
+{
+    /// <summary>
+    ///     Parameters for <see cref="EasyPost.Services.ShipmentService.Create"/> API calls.
+    /// </summary>
+    public class Create : Parameters, IShipmentParameter
+    {
+        #region Request Parameters
+
+        [TopLevelRequestParameter(Necessity.Optional, "carbon_offset")]
+        public bool AddCarbonOffset { get; set; }
```

Comment:
```suggestion
        public bool? AddCarbonOffset { get; set; }
```

Missing `?` on some of these optional properties. This applies to other places.

Reply:
Bool doesn't need to be explicitly nullable. Will default to "false" (or whatever default value we set it to) and always be included.

Or we could make it nullable, in which case it won't be included in the JSON body unless explicitly set.

Thoughts @Justintime50 ?

Reply:
Why is this called `AddCarbonOffset` instead of just `CarbonOffset`?

We determined we'd always send this when we first implemented it (meaning it's always true or false), let's not change directions now.